### PR TITLE
Add chart source and document it

### PR DESCRIPTION
**Pull request by honzajavorek**
#### :rocket: Why this change?

Even though it is an obscure format, the source should still be available to the public.

#### :memo: Related issues and Pull Requests

Follow-up to https://github.com/apiaryio/dredd/pull/1402 and https://github.com/apiaryio/dredd/pull/1416

#### :white_check_mark: What didn't I forget?

- [x] To write docs
- [ ] To write tests
- [x] To put [Conventional Changelog](https://dredd.org/en/latest/internals.html#sem-rel) prefixes in front of all my commits and run `npm run lint`

### CODE REVIEW

**Comment by honzajavorek** [comment]
@artem-zakharchenko Could you look at this one, please?

File: docs/internals.rst [hunk 1]
```diff
@@ -291,6 +291,19 @@ Writing documentation
 -  Be consistent
 
 
+.. _images:
+
+Images
+~~~~~~
+
+Images are in the ``docs/_static/images`` directory. For images exported from sophisticated graphic formats, the source file should be commited to Git and placed in the same directory, with the same basename, just with different extension.
```
Comment (artem-zakharchenko) [inline docs/internals.rst]:
```diff
- exported from sophisticated graphic formats
+ exported in sophisticated graphic formats
```

If you are speaking about the format of the exported images, then it's _in_.

Comment (artem-zakharchenko) [inline docs/internals.rst]:
```diff
- commited
+ committed
```

**Comment by honzajavorek** [comment]
@artem-zakharchenko addressed, thanks!

**Comment by honzajavorek** [comment]
Tests failing because of this https://github.com/apiaryio/dredd/pull/1427

**Comment by honzajavorek** [comment]
Rebased against master, will merge if tests are passing.

**Pull request merged**